sdk/PrebidDemandSDKLoaders/DemandSDKLoaders/FacebookAudienceNetwork/PBFacebookBannerAdLoader.m

```diff
@@ -0,0 +1,123 @@
+/*   Copyright 2017 APPNEXUS INC
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "PBFacebookBannerAdLoader.h"
+
+#import "PBConstants.h"
+
+struct FBAdSize {
+    CGSize size;
+};
+
+@implementation PBFacebookBannerAdLoader
+
+- (void)loadAd:(NSDictionary *)info {
+	NSString *bidPayload = (NSString *)info[@"adm"];
+	CGFloat height = [(NSString *)info[@"height"] floatValue];
+	CGSize adSize = CGSizeMake(-1, height);
```

Comment:
If size is 320x50 then use 320x50, if height is in 50, 90 or 250 then use their constants, other sizes  just do not instantiate.